vaadinfrontend/frontend/themes/datamanager/components/spreadsheet.css

```diff
@@ -0,0 +1,21 @@
+vaadin-combo-box-overlay::part(content) {
+  height: 30rem;
+  width: 20rem;
+  max-height: 30vh;
+}
+
+vaadin-combo-box-overlay::part(overlay) {
+  height: 30rem;
+  width: 20rem;
+  max-height: 30vh;
+}
```

Comment:
This will change the overlay for all comboboxes independent of where they are used. Is this intended? If so please move this to `vaadin-custom.css` .
If not you can add a classname before this to make it specific ala: 

```
.spreadsheet-X-Y vaadin-combo-box-overlay::part(content) {
}
```